docs/gcp_workload_identity.md

```diff
@@ -46,14 +46,22 @@ With Workload Identity, the type of the credentials is `external_account`. `audi
 
 ### Steps to install an OpenShift Cluster with Workload Identity
 
-1. Set `$RELEASE_IMAGE` to point to a sufficiently new OpenShift release.
+1. Set the variable `$RELEASE_IMAGE`
+
+   `$RELEASE_IMAGE` should be a sufficiently new OpenShift release image that you want to deploy in your cluster.
```

Comment:
This is ambiguous. By `sufficiently new` means how new? We have [support matrix](https://github.com/openshift/cloud-credential-operator#support-matrix) for that. It would be great if we can mention the exact version here or some kind of reference to the support matrix